ui/web/src/views/ContactDetails.vue

```diff
@@ -88,52 +89,86 @@
           </v-flex>
         </v-layout>
       </v-flex>
-      <v-flex v-if="contact.tags.length > 0" px-2>
+      <v-flex v-if="contact.tags != 'Empty'" px-2>
         <v-flex xs1>
           <span class="field-label">Tags</span>
         </v-flex>
         <v-flex xs2>
-          <ul>
-            <li v-for="(tag, index) in contact.tags" :key="index">{{tag}}</li>
-          </ul>
+          {{contact.tags}}
         </v-flex>
       </v-flex>
     </v-layout>
   </v-container>
 </template>
 
 <script>
+import RestServices from "../services/RestServices";
+
 export default {
   name: "ContactDetails",
-
-  // Mocked data for later API integration
   data: () => ({
     contact: {
-      name: "Carlos Andres Saavedra Regalado",
-      company: "Arkusnexus",
-      email: "csaavedra@arkusnexus.com",
-      mobile_phone: "(664) 111-1111",
-      home_phone: "(664) 222-2222",
-      birthday: "01/01/1990",
-      office_phone: "(664) 111-1111",
-      fax: "(664) 222-2222",
-      tags: []
+      name: "",
+      company: "",
+      email: "",
+      mobile_phone: "",
+      home_phone: "",
+      birthday: "",
+      office_phone: "",
+      fax: "",
+      tags: ""
+    }
+  }),
+  mounted() {
+    const restServices = new RestServices();
+
+    restServices.getContact(this.$route.params.id).then(res => {
+      const empty = "Empty";
+      const contact = res.data;
+
+      this.contact.name = `${contact.first_name} ${contact.last_name}`;
```

Comment:
Use object declaration
this.contact = {
    fax: contact.fax,
    email: contact.email
}

Don't use "Empty" as a string in the object, display "Empty" string if needed in the template directly not on the contact object

Reply:
Fixed and changed